all.tf

```
@@ -76,11 +76,11 @@ data "aws_iam_policy_document" "all" {
 }
 
 module "all_label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=0.6.2"
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=0.14.1"
   namespace  = "${var.namespace}"
```

now, it's not necessary brackets
```suggestion
  namespace  = "var.namespace"
```

---

should be fixed in all code :D

---

could be fixed throughout the code

---

Done. Fixed the rest of the code up as well.

---

lgtm now :)